### PR TITLE
fix: UI improvement

### PR DESCRIPTION
**Pull request by MohammedMokresh**
----
#### PR Submission Checklist for internal contributors

- The **PR Title**
    - [ ] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
    - [ ] contains a reference JIRA issue number like `SQPIT-764`
    - [ ] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
    - [ ] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

### Issues

links and mentioned color are wrong, asset name has the extension with it 

### Solutions

update the color, and show only the name for the file 
### Dependencies (Optional)

_If there are some other pull requests related to this one (e.g. new releases of frameworks), specify them here._

Needs releases with:

- [ ] GitHub link to other pull request

#### Test Coverage (Optional)

- [ ] I have added automated test to this contribution

#### How to Test

_Briefly describe how this change was tested and if applicable the exact steps taken to verify that it works as expected._

### Notes (Optional)

_Specify here any other facts that you think are important for this issue._

### Attachments (Optional)

_Attachments like images, videos, etc. (drag and drop in the text box)_ 

----
#### PR Post Submission Checklist for internal contributors (Optional)

- [ ] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

- [ ] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

**Comment by AndroidBob** [comment]
Build [2](https://10.10.124.134/job/wire-android-reloaded/job/PR-1446/2/) **failed**.

**Comment by AndroidBob** [comment]
Build [3](https://10.10.124.134/job/wire-android-reloaded/job/PR-1446/3/) **failed**.

File: app/src/main/kotlin/com/wire/android/mapper/MessageContentMapper.kt [hunk 1]
```diff
@@ -239,7 +239,7 @@ class MessageContentMapper @Inject constructor(
                 // It's a generic Asset Message so let's not download it yet
                 else -> {
                     UIMessageContent.AssetMessage(
-                        assetName = name ?: "",
+                        assetName = name?.split(".")?.first() ?: "",
```
Comment (gongracr) [inline app/src/main/kotlin/com/wire/android/mapper/MessageContentMapper.kt:242]: We actually need the extension on the asset name as I believe it is used in some other parts of the code, so I'd be careful when taking it out

**Comment by gongracr** [comment]
Changes were already present on `develop` branch and not top priority for RC.

**Pull request closed without merging**